Resources/Prototypes/_Mono/Shipyard/triage.yml

```
@@ -0,0 +1,54 @@
+# SPDX-FileCopyrightText: 2023 IHAN <IHAN>
+# SPDX-FileCopyrightText: 2023 Kennedy
+# SPDX-FileCopyrightText: 2023 RealIHaveANameOfficial
+# SPDX-FileCopyrightText: 2024 Dvir
+# SPDX-FileCopyrightText: 2024 Long YM
+# SPDX-FileCopyrightText: 2024 Maxtone
+# SPDX-FileCopyrightText: 2024 Whatstone
+# SPDX-FileCopyrightText: 2025 Primus
+# SPDX-FileCopyrightText: 2025 kasature90
+# SPDX-FileCopyrightText: 2025 starch
+#
+# SPDX-License-Identifier: MPL-2.0
+
+# Author Info
+# GitHub: Primusirius
+# Discord: Primusirius
+
+# Shuttle Notes:
+#
+- type: vessel
+  id: Triage
+  parent: BaseVessel
+  name: Triage
+  description: A extremely compact medical ship that packs a punch.
+  price: 42000
+  category: Small
+  group: Medical
+  shuttlePath: /Maps/_Mono/Shuttles/triage.yml
+  guidebookPage: null
+  class:
+  - Medical
+  engine:
+  - APU/RTG
```

it should be done like that.
```suggestion
  engine:
  - APU
  - RTG
```